gmime/part.go

```diff
@@ -119,3 +120,10 @@ func (p *Part) ContentID() string {
 func (p *Part) asGMimeObject() *C.GMimeObject {
 	return p.gmimePart
 }
+
+// String returns content as a string
+func (p *Part) String() string {
```

Comment:
can you add a test? also you probably want to test for memory leak just to make sure the logic to free memory is working correctly. you can parse the mime and run this function in a loop and monitor the memory usage.